packages/web/src/pages/profile-page/MobileServerProfilePage.tsx

```diff
@@ -0,0 +1,213 @@
+
+import styles from './components/mobile/ProfileHeader.module.css'
+
+const messages = {
+  tracks: 'Tracks',
+  followers: 'Followers',
+  following: 'Following',
+  playlists: 'Playlists',
+  showMore: 'Show More',
+  showLess: 'Show Less',
+  editProfile: 'Edit Profile',
+  profilePicAltText: 'User Profile Picture'
+}
+
+export type OwnProps = {
+  // userId: ID
+  userId: any
+}
+
+export const ServerProfilePage = ({ userId }: OwnProps) => {
+  const user = useSelector((state) => getUser(state, { id: userId }))
+  const [isDescriptionMinimized, setIsDescriptionMinimized] = useState(true)
+  if (!user) return null
+
+  const {
+    bio,
+    name,
+    handle,
+    cover_photo,
+    profile_picture,
+    track_count,
+    playlist_count,
+    follower_count,
+    followee_count,
+    donation,
+    website
+  } = user
+
+  const isArtist = track_count > 0
+  const hasEllipsis = donation || website
+
+  return (
+    <Flex direction='column' backgroundColor='default'>
+      <Flex
+        w='100%'
+        h={96}
+        direction='column'
+        justifyContent='center'
+        alignItems='center'
+        css={{ overflow: 'hidden' }}
+      >
+        {/* @ts-ignore */}
+        <Box as='img' src={cover_photo!} w='100%' />
+        {isArtist && !user.is_deactivated ? (
+          <BadgeArtist className={styles.badgeArtist} />
+        ) : null}
+      </Flex>
+      <Box className={styles.profilePictureWrapper}>
+        <Box
+          as='img'
+          // @ts-ignore
+          src={profile_picture!}
+          h='100%'
+          w='100%'
+          borderRadius='m'
+        />
+      </Box>
+      <div className={styles.artistInfo}>
+        <Flex justifyContent='space-between'>
+          <Flex direction='column' gap='xs'>
+            <Flex>
+              <Text
+                color='default'
+                variant='title'
+                size='l'
+                css={{ lineHeight: '22px' }}
+              >
+                {name}
+              </Text>
+            </Flex>
+            <Box css={{ textAlign: 'left', marginBottom: 14 }}>
+              <Box className={styles.artistHandle}>{handle}</Box>
+            </Box>
+          </Flex>
+        </Flex>
+
+        <Flex inline justifyContent='flex-start' w='100%' mb='s' gap='l'>
+          <Flex css={{ gap: 3 }}>
+            <Text
+              color='default'
+              variant='title'
+              size='s'
+              strength='strong'
+              css={{ lineHeight: 'normal' }}
+            >
+              {formatCount(isArtist ? track_count : playlist_count)}
+            </Text>
+            <Text
+              color='subdued'
+              variant='title'
+              size='s'
+              strength='weak'
+              css={{ lineHeight: 'normal' }}
+            >
+              {isArtist ? messages.tracks : messages.playlists}
+            </Text>
+          </Flex>
+          <Flex css={{ gap: 3 }}>
```

Comment:
funny having to use harmony on existing designs that dont follow spec 🥲

Reply:
😢